arkane/ess/gaussianTest.py

```diff
@@ -57,6 +57,18 @@ def setUpClass(cls):
         """
         cls.data_path = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'data', 'gaussian')
 
+    def test_check_for_errors(self):
+        """
+        Uses Gaussian log files that had various errors
+        to test if errors are properly parsed.
+        """
+        with self.assertRaises(LogError):
+            GaussianLog(os.path.join(self.data_path, 'l913.out'))
+        with self.assertRaises(LogError):
+            GaussianLog(os.path.join(self.data_path, 'l9999.out'))
+        with self.assertRaises(LogError):
+            log = GaussianLog(os.path.join(self.data_path, 'error_termination.out'))
+
```

Comment:
Better to also add a check for cases when `check_for_errors = False`. Same for other Log classes.

Reply:
Sorry help me understand this comment. What would I check for if `check_for_errors = False`?

Reply:
I think this is fine. I was going to suggest something similar as well, but then I realized that if `check_for_errors = False` then the program will just run, so nothing to check